_data/projects/powershell-docker.yml

```
@@ -0,0 +1,10 @@
+name: PowerShell Docker Images
+desc: The docker images PowerShell releases, both official and community.
+site: https://github.com/powershell/powershell-docker
+tags:
+  - powershell
+  - docker
+  - oss
+upforgrabs:
+  name: up for grabs
```

@TravisEz13 As per your Labe link and name of label in your Repo, this should be 'up-for-grabs'
```suggestion
  name: up-for-grabs
```